### PR TITLE
feat(fig): add fig-team cli generators

### PR DESCRIPTION
**Pull request by fedeci**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

**Comment by grant0417** [comment]
Just a note on the generators, most of the CLI commands can actually take a `--format json` which would be easier to use than spiting on newlines and gives more data.

**Comment by fedeci** [comment]
@SeparateRecords @grant0417 @mschrage this is ready, please read the comments I left for future automatic updates of this spec!

**Comment by fedeci** [comment]
Fig's autocomplete is great to review CLI tools too! If the spec gets too complicated it may be worth to simplify the CLI😃

**Review by grant0417 — APPROVED** [review]

File: src/fig/1.0.0.ts [hunk 1]
```diff
@@ -2227,114 +2232,124 @@ versions["1.4.0"] = {
...
-          options: [
-            {
-              name: "--version",
-              description: "Print version information",
-            },
+      args: {
+        name: "team",
+        generators: teamsGenerators,
+        isOptional: true,
+        loadSpec: {
```
[the remark this answers is not in the document]
Comment (fedeci) [reply src/fig/1.0.0.ts:2239]: Ping 🙂

File: src/fig/1.0.0.ts [hunk 2]
```diff
@@ -2176,7 +2180,7 @@ versions["1.4.0"] = {
                   description: "The team namespace to revoke the token for",
                   args: {
                     name: "team",
-                    isOptional: true,
```
[the remark this answers is not in the document]
Comment (fedeci) [reply src/fig/1.0.0.ts:2179]: Ping 🙂

**Pull request merged**